client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -453,10 +451,11 @@ const PluginMeta = React.createClass( {
 							</div>
 							{ ! isEmpty( actionLinks ) &&
 								<div className="plugin-meta__action-links">
-									{ Object.keys( actionLinks ).map( linkTitle => (
+									{ Object.keys( actionLinks ).map( ( linkTitle, index ) => (
 										<Button compact icon
 											href={ actionLinks[ linkTitle ] }
 											target="_blank"
+											key={ 'action-link-' + index }
```

Comment:
I think the `linkTitle`s are not stable enough for use there. They can vary based on the admin language of the Jetpack site (see https://github.com/Automattic/jetpack/issues/7127 for details), so I'd probably go with the index here too.